rasa_core/train.py

```diff
@@ -139,7 +140,7 @@ def create_argument_parser():
             '--fallback_action_name',
             type=str,
             default=DEFAULT_FALLBACK_ACTION,
-            help="When a fallback is triggered (e.g. because the ML prediction "
+            help="When a fallback is triggered (e.g. because the ML prediction"
```

Comment:
please add a space after `prediction` otherwise it's `because of the ML predictionis of low confidence`

Reply:
I think it was crossing the 80 character limit

Reply:
then wrap it differently, e.g. 
```python
help="When a fallback is triggered (e.g. because the ML "
                 "prediction is of low confidence) this is the name of the "
                 "action that will get triggered instead."
```

There is also a typo in line 144 (tje -> the). I know it was not caused by you, but would be cool if you just fix it together with the missing space.

Reply:
okay no problem.